tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -26,6 +26,11 @@ public static partial class Tags
         /// </summary>
         public const string Version = "version";
 
+        /// <summary>
+        /// A boolean indicating whether APM tracing is disabled when set to 0.
+        /// </summary>
+        internal const string ApmEnabled = "_dd.apm.enabled";
```

Comment:
If this is a numeric tag, this constant should probably go in `Metrics.cs`. Unless we're adding all tags (string and numeric) into this file going forwards, but that's not a thing AFAIK.

Reply:
Changed in 27ab78bc6db0ea98c5f6fd5a0d20f9487fbe3a0b